mopidy_local/commands.py

```diff
@@ -143,19 +147,54 @@ def _check_tracks_in_library(
         return files_to_update, files_in_library
 
     def _find_files_to_scan(
-        self, *, media_dir, file_mtimes, files_in_library, excluded_file_exts
+        self,
+        *,
+        media_dir,
+        file_mtimes,
+        files_in_library,
+        included_file_exts,
+        excluded_file_exts,
     ):
         files_to_update = set()
 
+        def _is_hidden_file(relative_path, file_uri):
+            if any(p.startswith(".") for p in relative_path.parts):
+                logger.debug(f"Skipped {file_uri}: Hidden directory/file")
+                return True
+            else:
+                return False
+
+        def _extension_filters(
+            relative_path, file_uri, included_file_exts, excluded_file_exts
+        ):
+            if included_file_exts:
+                if relative_path.suffix.lower() in included_file_exts:
+                    logger.debug(f"Added {file_uri}: File extension on included list")
+                    return True
+                else:
+                    logger.debug(
+                        f"Skipped {file_uri}: File extension not on included list"
+                    )
+                    return False
+            else:
+                if relative_path.suffix.lower() in excluded_file_exts:
+                    logger.debug(f"Skipped {file_uri}: File extension excluded")
+                    return False
+                else:
+                    logger.debug(f"Included {file_uri}: File extension not excluded")
```

Comment:
Nitpick: Change these debug lines to be more consistently worded. e.g. 
`"Added {file_uri}: File extension not on excluded list"`

And similarly for the one just above. You could even remove this and the other “Added" debug messsge, it's the skipped files that are going to be interesting, the rest is arguably noise.

Reply:
OK, got it.

Since this is a debug rather than just an info log, I wouldn't mind keeping the "added" entries as well. There were a few times when I got confused with various permutations of negations, and it was useful to see explicitly which branch was executing. But up to you!